src/openeo_grass_gis_driver/collections.py

```diff
@@ -7,6 +7,8 @@
 from openeo_grass_gis_driver.actinia_processing.config import Config
 from openeo_grass_gis_driver.models.collection_schemas import \
      Collection, CollectionEntry
+# from openeo_grass_gis_driver.utils.logging import log
```

Comment:
Remove commented code?

Reply:
Yes - I have to admit I was a bit lazy because already preparing the logging locally. But you are right + I removed it now